platform/osgi/platform-osgi-configadmin/src/main/java/org/codice/felix/cm/file/EncryptingPersistenceManager.java

```diff
@@ -162,10 +167,16 @@ synchronized String encrypt(String plainTextValue) {
       if (initFailureMessage != null) {
         alertSystem(initFailureMessage, initException);
       }
+      if (isBlank(plainTextValue)) {
+        LOGGER.debug(
+            "Failed to encrypt value of %s, because it was null or blank.", plainTextValue);
+        return plainTextValue;
+      }
+
       try {
         return crypter.encrypt(plainTextValue);
       } catch (CrypterException e) {
-        LOGGER.warn(String.format("Failed to encrypt to bundle cache. %s", e.getCause()));
+        LOGGER.warn(String.format("Failed to encrypt to bundle cache. %s", e.getMessage()));
```

Comment:
this should be parameterized

Reply:
Forgot I could use `{}` in the Logger. Changing now 🎉